deposit/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "deposit"
+version = "0.0.0"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/eigerco/substrate-move.git", subdir = "language/move-stdlib", rev = "main" }
+
+[addresses]
+std = "0x1"
```

Comment:
please elaborate how this is auto resolve after merge?

Reply:
this approach with declaring `std` is the one present in all [official repo](https://github.com/move-language/move/blob/ea70797099baea64f05194a918cebd69ed02b285/language/evm/stdlib/Move.toml#L6)'s examples I've seen and it might be confusing for consumers of our solution to understand why it works like that, but ok.